app/storage.js

```diff
@@ -34,7 +34,7 @@ const getFileStream = async (filename) => {
   containersInitialised ?? await initialiseContainers()
   const client = container.getBlockBlobClient(`${config.folder}/${filename}`)
   try {
-    return await client.download(0)
+    return await client.download()

```

Comment:
Bit of a hidden param so removed.

0 refers to what offset to read from, 0 being the start.

Providing no param defaults to 0 so not needed.